### PR TITLE
fix: change docs endpoint

### PR DESCRIPTION
**Pull request by dimgogos**

### CODE REVIEW

File: charts/drax/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: drax
 description: Accelleran's dRAX helm chart
 type: application
-version: 7.0.0-rc.5
+version: 7.0.0-rc.6
```
Comment (ben-foxmoore) [inline charts/drax/Chart.yaml]: I think we should leave this out of the PR, as it avoids conflicts between different PRs that might be merged before or after each other. Once the PR is merged (and possibly others are too), we can tag a new RC.

**Review by ben-foxmoore — COMMENTED** [review]
The status check failed to pull the `dash-front-back-end:4.8.0` image. I guess it hadn't finished building. Looks like it's done now.

**Review by Wielewout — COMMENTED** [review]
I've triggered the workflow again now that the docker image exists.

As for the conventional commits: I maybe would describe this rather a fix, maybe even a feature? Depends on whether we describe the previous behaviour as broken. `fix` and `feat` are the types that trigger new releases, which we obviously would want for an app version bump. At some point the version bumps should get suggested by renovate by the way, making these types of PRs unnecessary :smile:

Also as Ben mentioned, the chart version bumps (except the nested ones for now) should be done separately (as in separate commits and even as a separate PR). This is maybe not strictly necessary from a functionality point of view, but more to keep commits for release notes a bit more separated (as there will be links to the actual code).

File: charts/drax/Chart.yaml (hunk 1 above)
Comment (dimgogos) [reply charts/drax/Chart.yaml]: agreed

**Comment by dimgogos** [comment]
> I've triggered the workflow again now that the docker image exists.
> 
> As for the conventional commits: I maybe would describe this rather a fix, maybe even a feature? Depends on whether we describe the previous behaviour as broken. `fix` and `feat` are the types that trigger new releases, which we obviously would want for an app version bump. At some point the version bumps should get suggested by renovate by the way, making these types of PRs unnecessary 😄
> 
> Also as Ben mentioned, the chart version bumps (except the nested ones for now) should be done separately (as in separate commits and even as a separate PR). This is maybe not strictly necessary from a functionality point of view, but more to keep commits for release notes a bit more separated (as there will be links to the actual code).

ok I will tag it as a fix then

**Review by Wielewout — APPROVED** [review]

**Comment by ben-foxmoore** [comment]
@dimgogos sorry - I should have merged this first. I'll resolve the conflict (dashboard version is now 5.0.0) and merge it.

**Pull request merged**